<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <widget class="QLineEdit" name="ServerIPlineEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ServerPortlineEdit">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>28</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ListenpushButton">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>27</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ServerReceDatatextEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>85</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>56</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ReceiveData:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ServerSendDatalineEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>365</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerSendDatapushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>391</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SendData</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>347</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SendData:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IPAddress:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ServerCheckBox">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>59</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LogRecord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerDelpushButton">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>56</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ClearData</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ServerHexcheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>59</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
